src/Castle.Core.Tests/DynamicProxy.Tests/DefaultInterfaceMembersTestCase.cs

```diff
@@ -397,12 +397,12 @@ public void Can_proxy_interface_with_static_method()
 		}
 
 #if NET7_0_OR_GREATER
-		[Test]
-		[Ignore("Support for static abstract interface members has not yet been implemented.")]
-		public void Can_proxy_interface_with_static_abstract_method()
-		{
-			_ = generator.CreateInterfaceProxyWithoutTarget<IHaveStaticAbstractMethod>();
-		}
+		//[Test]
+		//[Ignore("Support for static abstract interface members has not yet been implemented.")]
+		//public void Can_proxy_interface_with_static_abstract_method()
+		//{
+		//	_ = generator.CreateInterfaceProxyWithoutTarget<IHaveStaticAbstractMethod>();
+		//}
```

Comment:
I'm not sure I understand this change. Why comment out the test? Does `[Ignore]` no longer work?

Reply:
I see.

I think it would be good to keep the test for its intended purpose as an implementation reminder, but it also shouldn't hold you back at this moment.

How about we only comment out the erroneous bit and leave a note for later?:

```suggestion
		[Test]
		[Ignore("Support for static abstract interface members has not yet been implemented.")]
		public void Can_proxy_interface_with_static_abstract_method()
		{
			// NOTE: uncommented because of compilation error CS8920, to be reviewed when we implement support for static abstract interface members
			// _ = generator.CreateInterfaceProxyWithoutTarget<IHaveStaticAbstractMethod>();
		}
```

(If that should bring up yet another complication, we'll do it your way and comment out the whole thing... for now.)

Reply:
done